logistics_planning_invoicing/wizards/logistics_schedule_existing_account_move_wizard.py

```diff
@@ -0,0 +1,34 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class LogisticsScheduleExistingAccountMove(models.TransientModel):
+    _name = 'logistics.schedule.existing.account.move.wizard'
+    _description = "Logistics Schedule Add Existing Invoice wizard"
+
+    logistics_schedule_id = fields.Many2one('logistics.schedule', readonly=True)
+    product_id = fields.Many2one('product.product', related='logistics_schedule_id.product_id')
+    carrier_id = fields.Many2one('res.partner', related='logistics_schedule_id.carrier_id')
+    invoice_line_id = fields.Many2one('account.move.line', string="Invoice Line")
+    domain_invoice_line_id = fields.Many2many('account.move.line', compute='_compute_domain_invoice_line_id')
+
+    def _prepare_domain_invoice_line_id(self):
+        product_id = self.logistics_schedule_id._get_invoice_product()
+        if not product_id:
+            raise ValidationError(_("It is necessary to configure the default products of the logistics planning"))
+        return [('move_id.partner_id', '=', self.carrier_id.id), ('product_id', '=', product_id.id)]
```

Comment:
I miss a default condition: only select those move lines that actually are invoice lines (see https://github.com/odoo/odoo/blob/4cc086d330b73514fbc64a7fcf22d8a7a9f1b691/addons/account/models/account_move.py#L216)